torch/ao/quantization/fx/prepare.py

```diff
@@ -475,6 +512,43 @@ def _get_arg_target_dtype_as_output(
     # TODO: should support is_dynamic here as well
     return output_act_dtype
 
+def _populate_arg_to_ctr_for_args(
+    arg_or_args: Union[Argument, Tuple[Argument, ...]],
+    input_obs_or_fq_ctr_for_args: Optional[Tuple[Any]],
+    arg_to_ctr: Dict[Argument, Any],
+) -> None:
+    if not isinstance(input_obs_or_fq_ctr_for_args, tuple):
+        # if input_obs_or_fq_ctr_for_args is a single element,
+        # use the same obs_or_fq_ctr for all args
```

Comment:
This is fine for now but I dont think we want to provide API where you use single observer that is used for all args. Just be explicit and require each arg's observer be specified. I think that what we agreed for prepare's interface?

Reply:
yes, this is for bc purposes, it is not recommended for new users to use this

Reply:
So I think we will need sort of something like:
Old_API -> populate_args_to_ctr this breaks without kwargs"
New_API -> something_else_which_provides_no_way_for_kwargs"
"populate_args_to_ctr this breaks without kwargs"  -> "rest of prepare"
"something else which provides no way for kwards" -> "rest of prepare"